API扩展.lua

```diff
@@ -153,3 +153,38 @@ if 
         return table.unpack(result)
     end
 end
+
+if Base.IsHash == nil then
+    function Base.IsHash(hash, x, y, width, height, thershold)
+        thershold = thershold or 10;
+        return (Base.ImageHashContrast(Base.ImageHash(x, y, width, height), hash) < thershold);
+    end
+end
+
+if Base.WaitHash == nil then
+    function Base.WaitHash(hash, x, y, width, height, thershold, interval, duration)
```

Comment:
That's right, but will it be neater if we have to write a Wait version for every base API call we want to wait for? In fact there are 2 versions of `ImageHash` and 4 versions of `IsColor` in the base API. Should we write a Wait version for each one of them? I can hardly see this being neater in the long run.